app/src/main/res/layout/widget_media_controls_configure.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Could we get a screenshot of the configure screen?

Reply:
Added to the PR description.